src/styles/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+
+  &-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+    background-image: url("../images/star-active.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
```

Comment:
```suggestion
  &__star:not(:last-child) {
    margin-right: 4px;
```
There is no need to duplicate all properties. 
Also, styles for modifiers are absent
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/931494a4-fdeb-40f6-85c4-0b869d7788f0)